Help-Rooms.md

```diff
@@ -14,7 +14,7 @@ _Choose the right room for your questions!_
 
 See also: :earth_asia: [**Language Specific**](#language-specific-rooms-) **|** :satellite: [**Technology**](#technology-based-rooms-) **|** :crystal_ball: [Miscellaneous](#miscellaneous-rooms-)
 
-[**`back to top`⮭**](#freecodecamp-chat-rooms)
+[**`back to top`&#8593;*](#freecodecamp-chat-rooms)
```

Comment:
Please use the symbol and add `**` after the symbol. Currently, the Markdown is invalid.